recipes/basemap-data-hires/bld.bat

```diff
@@ -0,0 +1,2 @@
+copy %SRC_DIR%\lib\mpl_toolkits\basemap\data\*_h.dat %LIBRARY_PREFIX%\share\basemap || exit 1
+copy %SRC_DIR%\lib\mpl_toolkits\basemap\data\*_f.dat %LIBRARY_PREFIX%\share\basemap || exit 1
```

Comment:
I think it is OK to ship with the 'crude' and 'low' as the default data and force a separate to get 'intermediate', 'high' and 'full'. I am doing that then.